### PR TITLE
Patch radiant-mlhub-landcovernet tutorial

### PR DESCRIPTION
**Pull request by m-cappi**
Bug:
- Land Cover Net global dataset is no longer available

Fix:
- Use the Land Cover Net Africa dataset instead

Changes:
- Had to drop the use of `pystac.extensions.label.LabelExtension` in favor of accessing the labels as JSONs
- Made some content adjustments in order to reflect the current changes

### CODE REVIEW

File: tutorials/radiant-mlhub-landcovernet.ipynb [hunk 1]
```diff
@@ -151,41 +151,41 @@
    "source": [
     "### Finding Possible Land Cover Labels\n",
     "\n",
-    "Each label item within the collection has a property which lists all of the possible land cover types and which ones are present in each label item. The code below prints out which land cover types are present in the dataset and we will reference these later in the notebook when we filter downloads."
+    "Each collection item has a property which lists all of the possible land cover types in the dataset and which ones are present in the current item.\n",
```
Comment (TomAugspurger) [inline tutorials/radiant-mlhub-landcovernet.ipynb:154]:
```suggestion
    "Each item has a property which lists all of the possible land cover types in the dataset and which ones are present in the current item.\n",
```

**Review by TomAugspurger — COMMENTED** [review]
Thanks! Looks like a small linting issue at https://github.com/microsoft/PlanetaryComputerExamples/actions/runs/3676329910/jobs/6219056203#step:4:87.

The CONTRIBUTING.md file should have a note on getting pre-commit set up locally if you want.

File: tutorials/radiant-mlhub-landcovernet.ipynb (hunk 1 above)
Comment (m-cappi) [reply tutorials/radiant-mlhub-landcovernet.ipynb]: Thanks Tom! Applied what you suggested

**Comment by TomAugspurger** [comment]
Thanks @m-cappi!

**Pull request merged**